conf/zapi/cdot/9.8.0/snapshotpolicy.yaml

```diff
@@ -20,6 +21,7 @@ plugins:
   - LabelAgent:
       split:
         - policy_owner `-admin` scope,
+#  - SnapshotViolation
```

Comment:
I would have expected SnapshotViolation to come after SnapshotPolicy instead of before, both from a chronological and alphabetical point of view